packages/dds/tree/src/changeset/format.ts

```diff
@@ -12,8 +12,8 @@ import { JsonableTree } from "../tree";
 /**
  * Changeset that has may have been transposed (i.e., rebased and/or postbased).
  */
-export namespace Transposed {
-	export interface Transaction extends PeerChangeset {
+export namespace ITransposed {
+	export interface Transaction extends IPeerChangeset {
 		/**
```

Comment:
Did you mean to add the 'I' prefix to the Transaction interface instead of the Transposed namespace?

Reply:
Yes. Changed.